test/nodejs/match.js

```diff
@@ -68,6 +68,7 @@ test('match: match in Monaco with geometry compression', function(assert) {
 
 test('match: match in Monaco with speed annotations options', function(assert) {
     assert.plan(12);
+    // var osrm = new OSRM({path: data_path, max_locations_map_matching: 2);
```

Comment:
Remove comment - or should this test a limit of 2 locations?

Reply:
Ah forgot to push the removal of this comment. Pushing now!